tokamak_geqdsk.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+#
+
+import sys
+
+if len(sys.argv) != 2:
+    raise ValueError("Usage is {} geqdsk_file".format(sys.argv[0]))
+
+filename = sys.argv[1]
+
+from hypnotoad2 import tokamak
+
+with open(filename, 'rt') as fh:
+    eq = tokamak.read_geqdsk(fh)
+
+# Make the regions, identifying single/double null etc.
+    
+eq.makeRegions(psinorm_pf=0.95, psinorm_sol=1.1)
```

Comment:
We should read an input file to get the options to pass in here. I'd suggest copying `parseInputs()` from `torpex.py`, although we probably don't need multiple sections so can simplify a bit more. I'd suggest getting the grid filename from the input file too, in the interests of saving everything in an input file rather than depending on command line inputs.

Reply:
This is done now with a yaml input file